contracts/Removal.sol

```diff
@@ -499,6 +446,58 @@ contract Removal is
     return id.decodeRemovalIdV0();
   }
 
+  /**
+   * @dev Transfers `amount` tokens of token type `id` from `from` to `to`.
```

Comment:
Generally, if we're keeping documentation to a minimum, I'd prefer we rely on `@notice` over `@dev`. [From the natspec docs](https://docs.soliditylang.org/en/latest/natspec-format.html):

> @notice - Explain to an end user what this does
> @dev - Explain to a developer any extra details

I think this is our main goal for documentation at the moment is to help the auditors understand what all these functions do at a higher level, rather than explain more subtle implementation details. So @notice is going to be more appropriate, since its purpose it to help a reader understand what a function does as a baseline, and then @dev is where you put any details to help the reader understand how it works.

Reply:
I think it would be worth calling out that this is implementation of one `safeTransferFrom` in the ERC1155 interface.

Reply:
Sure, thanks, in that case a `@notice` definitely isn't super necessary, I just chose this function at random.